.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  pluto:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+      args:
+        # [Choice] Python version: 3, 3.8, 3.7, 3.6
+        VARIANT: 3.9
+        NODE_VERSION: 16
```

Comment:
I'm curious about this line, I thought we didn't use node and didn't need to install it but maybe I'm wrong?

Reply:
Nope I should remove this.